bundles/org.openhab.ui/web/src/components/widgets/system/oh-repeater.vue

```diff
@@ -83,6 +83,8 @@ export default {
         return this.$oh.api.get('/rest/items/' + this.config.itemOptions).then((i) => Promise.resolve((i.stateDescription) ? i.stateDescription.options : []))
       } else if (this.config.sourceType === 'itemCommandOptions') {
         return this.$oh.api.get('/rest/items/' + this.config.itemOptions).then((i) => Promise.resolve((i.commandDescription) ? i.commandDescription.commandOptions : []))
+      } else if (this.config.sourceType === 'rulesWithTags' && this.config.ruleTags) {
+        return this.$oh.api.get('/rest/rules?summary=true' + '&tags=' + this.config.ruleTags).then((r) => r.sort(compareRules))
```

Comment:
```suggestion
        return this.$oh.api.get('/rest/rules?summary=true' + '&tags=' + this.config.ruleTags).then((r) => Promise.resolve(r.sort(compareRules)))
```

This asyncComputed property returns a promise for all cases, so it should behave the same here.